lms/lmsweb/views.py

```diff
@@ -501,9 +512,17 @@ def _common_comments(exercise_id=None, user_id=None):
     Most common comments throughout all exercises.
     Filter by exercise id when specified.
     """
-    query = CommentText.filter(**{
-        CommentText.flake8_key.name: None,
-    }).select(CommentText.id, CommentText.text).join(Comment)
+    is_moderator_comments = (
+        (Comment.commenter.role == Role.get_staff_role().id)
```

Comment:
`Comment.commenter.role.is_staff`?

Reply:
It's not possible - it throws an error that says is_staff, is_manager etc. are not attributes of Role.